### DISCUSSION TITLE
https://ovosimages.ziggyai.online/raspbian/newest/   how specify the SID and WIFI PW to connect to

### DISCUSSION THREAD

**Discussion by geetee24** [OpenVoiceOS/ovos-buildroot#223] [Q&A] [answered]
how specify?

**Comment by JarbasAl** [comment, marked as answer]
connect to SSID "OVOS"
open page openvoiceos.com
enter your credentials

same as in picroft, just named OVOS instead of Mycroft

**Comment by geetee24** [comment]
i have no SID OVOS.

**Comment by geetee24** [comment]
again I give up.

Your answers to me are consistently incomplete and hence very frustrating.

I need to tell your image of my sid and my sid's pw.
